package.json

```diff
@@ -90,11 +90,13 @@
     "**/graphql-toolkit/lodash": "^4.17.15",
     "**/hoist-non-react-statics": "^3.3.2",
     "**/isomorphic-git/**/base64-js": "^1.2.1",
+    "**/istanbul-instrumenter-loader/schema-utils": "1.0.0",
```

Comment:
This selective dependency resolution forces schema-utils `0.3.0` to resolve to `1.0.0`. According to the [changelog](https://github.com/webpack/schema-utils/blob/master/CHANGELOG.md#100-2018-08-07), there are no breaking changes between these versions.